gpflow/kernels/mo_kernels.py

```diff
@@ -137,7 +148,7 @@ class LinearCoregionalization(IndependentLatent, Combination):
     Linear mixing of the latent GPs to form the output.
     """
     def __init__(self, kernels, W, name=None):
-        Combination.__init__(self, kernels, name)
+        super().__init__(output_dim=len(kernels), kernels=kernels, name=name)
```

Comment:
pass through `**kwargs`.

(also something that we should do for *all* kernels? :thinking: )

Reply:
all abstract classes, yes. maybe all kernels, subject to discussion.